requirements.txt

```diff
@@ -0,0 +1,9 @@
+# Install
+simplegeneric
```

Comment:
I think `future` should be added as a dependency as well.

Reply:
I hadd to tweak the examples in `docs/examples` a little, due to the new naming of the toplevel package (`etkdocking`), but it works just fine.